Add 3 documentation rules to Vodafone Station (#37957)
home-assistant/home-assistant.github.io: source/_integrations/vodafone_station.markdown

{% include integrations/config_flow.md %}

> {% configuration %}

Reviewer: ```suggestion
{% configuration_basic %}
```

  host:

>     required: true

Reviewer: ```suggestion
```

    description: The IP address of the Vodafone Station router.

>     type: string

Reviewer: ```suggestion
```

  username:

>     required: true

Reviewer: ```suggestion
```

    description: The username of the Vodafone Station router.

>     type: string

Reviewer: ```suggestion
```

  password:

>     required: true

Reviewer: ```suggestion
```

    description: The password of the Vodafone Station router.

>     type: string

Reviewer: ```suggestion
```

> {% endconfiguration %}

Reviewer: ```suggestion
{% endconfiguration_basic %}
```

## Integration options

It is possible to change some behaviors through the {% term integration %} options.
To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. On the **Vodafone Station** integration, select the cogwheel. Then select **Configure**.

- **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home".

> ## Removing the integration

Reviewer: I would move the removal section to the end of the docs as the task is done at the end of the integration life cycle.

Author: Done

This integration follows standard integration removal. No extra steps are required.

{% include integrations/remove_device_service.md %}

## Data updates

> This integration fetches data from the device every 30 seconds by default.

Reviewer: ```suggestion
This integration {% term polling polls %} data from the device every 30 seconds by default.
```
